components/Markdown/Markdown.css

```diff
@@ -47,8 +47,8 @@
 }
```

Comment:
I love all of this. The only tiny thing that came to mind was knocking back the font `color` very slightly, using a dark grey instead of a black to soften the text. I tried `grey-02` in light mode and it seemed nice!

Reply:
3.0 is the required contrast for link-in-text without addtional indicator e.g. underline

<img width="492" alt="Screenshot 2023-12-04 at 12 58 21" src="https://github.com/measuredco/measuredco-site/assets/98794/093cc5f3-ef05-40e9-9253-c685eb6834e3">

Moving the text to `grey-01` (or any other color than #000) does not meet this requirement:

<img width="489" alt="Screenshot 2023-12-04 at 13 01 15" src="https://github.com/measuredco/measuredco-site/assets/98794/47a8bba3-f14d-4f6e-8210-a184dc27139b">

So changing this would require either 1. underlining all links in copy, or 2. unpicking the whole color system, which we could maybe look at at some pint but right now do not want 😅